cloudferrylib/os/actions/check_quotas.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from novaclient import exceptions as nova_exceptions
+
+from cloudferrylib.base import exception as cf_exceptions
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import log
+from cloudferrylib.utils import proxy_client
+from cloudferrylib.utils import utils
+
+
+LOG = log.getLogger(__name__)
+
+
+class CheckQuotas(action.Action):
```

Comment:
Can you please specify which quotas tenant or user quotas.  Also add that check on API level.